apps/bfd-server/bfd-server-war/src/test/resources/endpoint-responses/v2/claimMcsRead.json

```diff
@@ -168,7 +169,7 @@
       "productOrService": {
```

Comment:
Can we add a diagnosis code to this line item so we can test how `diagnosisSequence` is displayed?